jetty-servlets/src/test/java/org/eclipse/jetty/servlets/CrossOriginFilterTest.java

```diff
@@ -146,6 +149,31 @@ public void testSimpleRequestWithMatchingWildcardOrigin() throws Exception
         Assert.assertTrue(response.contains("Vary"));
         Assert.assertTrue(latch.await(1, TimeUnit.SECONDS));
     }
+    
+    @Test
+    public void testSimpleRequestWithRegexMatchingOrigin() throws Exception
+    {
+        FilterHolder filterHolder = new FilterHolder(new CrossOriginFilter());
+        String origin = "http://subdomain.example.com";
+        filterHolder.setInitParameter(CrossOriginFilter.ALLOWED_ORIGINS_REGEX_PARAM, "https?://.*[.]example[.]com");
```

Comment:
Hi, I still doubt that allowing wildcards is a good idea as `whatever.example.com` would pass the filter. 
Refer to: https://github.com/eclipse/jetty.project/issues/1053